back/app/serializers/web_api/v1/custom_field_serializer.rb

```diff
@@ -12,6 +12,11 @@ class WebApi::V1::CustomFieldSerializer < WebApi::V1::BaseSerializer
     object.resource_type == 'User'
   }
 
+  attribute :constraints do |object|
+    @participation_method = Factory.instance.participation_method_for object.resource.participation_context
```

Comment:
This code will also fail for user fields

Reply:
Yes of course. Will Fix that now. Good proposal for the specs, had already started on the validation specs but wanted to get this to out to the front-end before completing them.